examples/web_agent/prompt_templates.py

```diff
@@ -0,0 +1,38 @@
+# This file contains examples of prompts to be used
+
+EXTRACTION_PROMPT = """
```

Comment:
You can remove lines 1-12 on this file since EXTRACTION_PROMPT is not used in this project.

Reply:
Done